hledger-lib/Hledger/Reports/MultiBalanceReport.hs

```diff
@@ -102,16 +104,17 @@ type ClippedAccountName = AccountName
 -- by the balance command (in multiperiod mode) and (via compoundBalanceReport)
 -- by the bs/cf/is commands.
 multiBalanceReport :: ReportSpec -> Journal -> MultiBalanceReport
-multiBalanceReport rspec j = multiBalanceReportWith rspec j (journalPriceOracle infer j)
+multiBalanceReport rspec j = multiBalanceReportWith rspec j (journalPriceOracle infer j) mempty
   where infer = infer_prices_ $ _rsReportOpts rspec
 
--- | A helper for multiBalanceReport. This one takes an extra argument,
--- a PriceOracle to be used for looking up market prices. Commands which
--- run multiple reports (bs etc.) can generate the price oracle just
--- once for efficiency, passing it to each report by calling this
--- function directly.
-multiBalanceReportWith :: ReportSpec -> Journal -> PriceOracle -> MultiBalanceReport
-multiBalanceReportWith rspec' j priceoracle = report
+-- | A helper for multiBalanceReport. This one takes some extra arguments,
+-- a 'PriceOracle' to be used for looking up market prices, and a set of
+-- 'AccountName's which should always be displayed in the final report.
+-- Commands which run multiple reports (bs etc.) can generate the price oracle
+-- just once for efficiency, passing it to each report by calling this function
+-- directly.
+multiBalanceReportWith :: ReportSpec -> Journal -> PriceOracle -> Set AccountName -> MultiBalanceReport
+multiBalanceReportWith rspec' j priceoracle specialaccts = report
```

Comment:
Sorry for the fragmentary review. How would you feel about renaming specialaccts to unelidableaccts everywhere, for a bit more clarity ?

Reply:
Sure thing.